playbooks/portals-get-versions.yml

```diff
@@ -22,7 +22,6 @@
           - git commit {{ portal_repo_commit }}
 
           - portal_repo_version: {{ portal_repo_version }}
-          - portal_skyd_version: {{ portal_skyd_version_used }}
```

Comment:
I would not remove reporting `skyd` version, but would make it analogous to reporting `accounts` version.
`accounts` version reports either selected branch or image.

Reply:
I added back support for "portal_skyd_version_used" in bd711e6647fc2fbd4032ca218176687e003960b8 but I am not sure we should have that since we wanted to drop "sia" and "accounts" special handling across ansible but maybe it's not time to do that right now.

Tested on xyz:

Case 1: no override, docker compose uses old branch model
<img width="446" alt="Screenshot 2022-08-05 at 15 06 55" src="https://user-images.githubusercontent.com/3755029/183084579-42c31011-d0b7-4f90-b2ba-ed9be4a04563.png">

Case 2: no override, docker compose uses docker image
<img width="481" alt="Screenshot 2022-08-05 at 15 07 02" src="https://user-images.githubusercontent.com/3755029/183084817-6b9df327-d639-4375-9f2f-dde988a0c887.png">

Case 3: override skyd with custom branch
<img width="486" alt="Screenshot 2022-08-05 at 15 07 10" src="https://user-images.githubusercontent.com/3755029/183084784-1b52fe4e-e301-4d0f-9a80-cd23c656e25c.png">

Case 4: override skyd with custom image
<img width="493" alt="Screenshot 2022-08-05 at 15 11 18" src="https://user-images.githubusercontent.com/3755029/183084637-1e63f1b2-104f-4bd5-bfbf-bc46d51473ce.png">